### ISSUE TITLE
Trying to reach MusicBox

### ISSUE THREAD

**Issue by mauli16** [woutervanwijk/Pi-MusicBox#506]
Hi, 

I know there is a closed thread with the same problem, but the issue is still there for me. I tested it on a raspberry 1 and on a raspberry 3 b+ with the newest RC-version. Musicbox runs fine for approximately 24h, but after that it's broken and only shows 'Trying to reach Musicbox'. 

My wifi is off  over night. Maybe that's the problem? I deactivated unnecessary plugins too. Don't know what I can do. The only thing I could do was powering off the pi to get it running again.  

Is this still a known and unsolved issue? I was a heavy soundcloud user on musicbox. Maybe it's related to the soundcloud support?

Many thanks in advance :)

**Comment by kingosticks** [comment]
I suggest you don't turn your wifi off for a night and check if that's the problem.

**Comment by mauli16** [comment]
Thanks for your answer @kingosticks. Unfortunately it's not related to the WIFI. WIFI was on the whole time and I only used TuneIn and Soundcloud, but it happened again. 

The mopidy.log looks like below. Please let me know, any other log would be helpful to find the bug.  

`2018-11-22 15:30:09,409 - WARNING  Element doesn't implement handling of this stream. Please file a bug.
2018-11-22 15:30:45,268 - WARNING  Element doesn't implement handling of this stream. Please file a bug.
2018-11-22 15:31:00,753 - WARNING  Element doesn't implement handling of this stream. Please file a bug.
2018-11-22 15:31:15,541 - WARNING  Element doesn't implement handling of this stream. Please file a bug.
2018-11-22 15:31:40,263 - WARNING  Element doesn't implement handling of this stream. Please file a bug.
2018-11-22 15:32:01,870 - WARNING  Element doesn't implement handling of this stream. Please file a bug.
2018-11-22 15:32:37,535 - WARNING  Element doesn't implement handling of this stream. Please file a bug.
2018-11-22 15:35:22,268 - WARNING  Element doesn't implement handling of this stream. Please file a bug.
2018-11-22 15:36:29,074 - WARNING  Element doesn't implement handling of this stream. Please file a bug.
2018-11-22 15:36:45,113 - WARNING  Element doesn't implement handling of this stream. Please file a bug.
2018-11-22 15:37:35,702 - WARNING  Element doesn't implement handling of this stream. Please file a bug.
2018-11-22 15:37:52,615 - WARNING  Element doesn't implement handling of this stream. Please file a bug.
2018-11-22 15:40:23,770 - WARNING  Element doesn't implement handling of this stream. Please file a bug.
2018-11-22 15:41:16,644 - WARNING  404 GET /favicon.ico (192.168.178.21) 11.27ms
2018-11-22 15:41:16,672 - WARNING  404 GET /favicon.ico (192.168.178.21) 1.87ms
2018-11-22 15:41:23,094 - WARNING  404 GET /settings/js/mopidy.min.js (192.168.178.21) 1.82ms
2018-11-22 15:41:23,099 - WARNING  404 GET /settings/js/simple.js (192.168.178.21) 1.71ms
2018-11-22 15:41:23,587 - WARNING  404 GET /settings/js/mopidy.min.js (192.168.178.21) 1.78ms
2018-11-22 15:41:23,726 - WARNING  404 GET /settings/js/simple.js (192.168.178.21) 1.75ms
2018-11-22 15:43:11,303 - WARNING  Element doesn't implement handling of this stream. Please file a bug.
2018-11-22 16:42:49,583 - WARNING  Element doesn't implement handling of this stream. Please file a bug.
2018-11-22 16:52:34,967 - WARNING  Element doesn't implement handling of this stream. Please file a bug.
2018-11-22 21:43:56,092 - WARNING  Element doesn't implement handling of this stream. Please file a bug.
2018-11-22 21:44:02,519 - WARNING  Element doesn't implement handling of this stream. Please file a bug.
2018-11-22 21:44:09,731 - WARNING  Element doesn't implement handling of this stream. Please file a bug.
2018-11-22 21:46:34,502 - WARNING  Element doesn't implement handling of this stream. Please file a bug.
2018-11-23 10:29:23,919 - WARNING  Element doesn't implement handling of this stream. Please file a bug.
2018-11-23 11:46:37,144 - WARNING  Element doesn't implement handling of this stream. Please file a bug.
2018-11-23 15:27:07,744 - WARNING  Element doesn't implement handling of this stream. Please file a bug.
2018-11-23 15:29:15,907 - WARNING  Element doesn't implement handling of this stream. Please file a bug.
2018-11-23 15:33:22,462 - WARNING  Element doesn't implement handling of this stream. Please file a bug.
2018-11-23 15:34:55,576 - WARNING  Element doesn't implement handling of this stream. Please file a bug.
2018-11-23 15:41:17,350 - WARNING  Element doesn't implement handling of this stream. Please file a bug.`

**Comment by mauli16** [comment]
just fyi, I'm running volumio on the same pi without any problems now. So it seems that musicbox is unstable here.

**Comment by kingosticks** [comment]
In what respect? Volumio image with Mopidy-MusicBox-Webclient installed?

**Comment by mauli16** [comment]
No, just standard volumio in version 2.502. I just thought it would be helpful to let you and the community know, that musicbox is unstable for me and it's not hardware or network related in this case.

**Comment by AurelFR** [comment]
if you turn off the router and turn again ,you will get a new ip .that s why you cannot connect ,try to search with a prg your new ip again ...

**Comment by marioqx** [comment]
I have seen that this post is compartively old, but I have the same problem.
I switch off Wifi over night.

Issue is, that pimusicbox does not automatically re-connect.

Solution:
ssh to the pi-musicbox and enter:

`cd /etc/ifplugd/action.d/

mv ifupdown ifupdown.original

cp /etc/wpa_supplicant/ifupdown.sh ./ifupdown`

Source: https://raspberrypi.stackexchange.com/questions/4120/how-to-automatically-reconnect-wifi

I hope this will be added to v0.8.
